src/constraints/scalaraffine.jl

```diff
@@ -35,7 +35,7 @@ end
 function add_linear_constraint(model::LinQuadOptimizer, func::Linear, set::IV)
     columns = [get_column(model, term.variable_index) for term in func.terms]
     coefficients = [term.coefficient for term in func.terms]
-    A = CSRMatrix{Float64}([1], columns, coefficients)
+    A = CSRMatrix{Float64}([1,length(coefficients)+1], columns, coefficients)
```

Comment:
Nit: spaces after `,` and around binary operators such as `+`. This occurs quite often in the changes below.